src/utils/createRequire.js

```diff
@@ -0,0 +1,4 @@
+export default (module, disableProcessing) => {
+  const loader = disableProcessing ? '-!' + module : module;
+  return `require (${JSON.stringify(loader)});`;
```

Comment:
It's used for user's input require also, so I decided to leave `JSON.stringify`.